Dockerfile

```diff
@@ -19,5 +20,13 @@ ENV TUNE_TCP=1 \
     net__ipv4__tcp_tw_reuse=1 \
     net__ipv4__tcp_max_orphans=0
 
+# DOCKER_IMAGES
+ENV TUNE_IMAGES=1 \
```

Comment:
I would prefer `TUNE_DOCKER` instead and leave the containers/images cleanup as inner options.